api/pom.xml

```diff
@@ -65,6 +65,16 @@
             <artifactId>argon2-jvm</artifactId>
             <version>2.4</version>
         </dependency>
+        <dependency>
+            <groupId>net.logicsquad</groupId>
+            <artifactId>nanocaptcha</artifactId>
+            <version>1.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.25</version>
```

Comment:
I have aligned the version as you suggested but if I try to move dependency to `dependencies/pom.xml` i get the following warning:
```
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.
SLF4J: Ignoring binding found at [jar:file:/home/ivanmrsulja/.m2/repository/org/slf4j/slf4j-log4j12/1.7.26/slf4j-log4j12-1.7.26.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.
```

Reply:
Because of this, tests that rely on log parsing for functionality verification fail, because there is no log output without the necessary provider. It was the sole reason I added this dependency here which solved the issue. If you have another solution in mind I would be glad to switch to it :slightly_smiling_face: .